kubernetes/namespaces/monitoring/alerts/alerts.d/django.yaml

```diff
@@ -11,7 +11,7 @@ groups:
           description: "Django is experiencing 5xx errors on {{ $labels.namespace }}/{{ $labels.job }}"
 
       - alert: DjangoLatencyElevated
-        expr: histogram_quantile(0.95, rate(django_http_requests_latency_seconds_by_view_method_bucket{view!="api:github-artifacts", view!="home:home", view!="content:tag"}[5m])) > 1.0
+        expr: histogram_quantile(0.95, rate(django_http_requests_latency_seconds_by_view_method_bucket{view!="api:github-artifacts", view!="api:github-webhook-filter", view!="home:home", view!="content:tag"}[5m])) > 1.0
```

Comment:
I don't know how prometheus alerting works at all

But, isn't `github-webhook-filter` just the name you assign to that view in site's api ? The actual url contains the webhook id and webhook token as path params

Will it then be able to know how to ignore it still ?

Or maybe there's some mechanism i'm not aware of to how these views are labeled/managed